system-variables.md

```diff
@@ -81,6 +81,13 @@ mysql> SELECT * FROM t1;
 - Default value: ON
 - Controls whether statements should automatically commit when not in an explicit transaction. See [Transaction Overview](/transaction-overview.md#autocommit) for more information.
 
+### datadir
+
+- Scope: NONE
+- Default value: /tmp/tidb
+- This variable indicates the location where data is stored.
```

Comment:
```suggestion
- This variable indicates the location where data is stored. This can be a local path or point to a PD server in case the data is stored on TiKV.
```